src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/UserServiceTest.kt

```diff
@@ -72,6 +79,23 @@ class UserServiceTest {
       verify(authApiService).findAzureUserByUsername(anyString())
       assertThat(user).isNull()
     }
+
+    @Nested
+    inner class MyRoles {
+      @Test
+      fun myRoles(): Unit = runBlocking {
```

Comment:
Not sure manage users needs runblocking yet

Reply:
Removed